WolvenKit.App/ViewModels/Tools/ProjectExplorerViewModel.cs

```diff
@@ -948,6 +960,98 @@ private void OpenInMlsb()
     private void SetupToolDefaults() =>
         ContentId = s_toolContentId; // Define a unique contentId for this toolwindow//BitmapImage bi = new BitmapImage();  // Define an icon for this toolwindow//bi.BeginInit();//bi.UriSource = new Uri("pack://application:,,/Resources/Media/Images/property-blue.png");//bi.EndInit();//IconSource = bi;
 
+    private void ReplacePathInProject(ResourcePath oldPath, ResourcePath newPath)
+    {
+        if (ActiveProject is null)
+        {
+            return;
+        }
+
+        _watcherService.IsSuspended = true;
+
+        var files = Directory.GetFiles(ActiveProject.ModDirectory, "*.*", SearchOption.AllDirectories);
+        Parallel.ForEach(files, file => ReplacePathInFile(file, oldPath, newPath));
+
+        _watcherService.IsSuspended = false;
+    }
+
+    private void ReplacePathInFile(string filePath, ResourcePath oldPath, ResourcePath newPath)
+    {
+        if (!File.Exists(filePath))
+        {
+            return;
+        }
+
+        if (oldPath == newPath)
+        {
+            return;
+        }
+
+        CR2WFile? cr2w;
+        var wasModified = false;
+
+        using (var fs = File.Open(filePath, FileMode.Open))
+        using (var cr = new CR2WReader(fs))
+        {
+            if (cr.ReadFile(out cr2w) != EFileReadErrorCodes.NoError)
+            {
+                return;
+            }
+
+            foreach (var result in cr2w!.FindType(typeof(IRedRef)))
+            {
+                if (result.Value is not IRedRef resourceReference || resourceReference.DepotPath != oldPath)
```

Comment:
I think this will not work with folders, you'd have to check StartsWith?

Reply:
Not sure why you commented on a outdated file

Reply:
because git sorted them in reverse temporal order and I was so confused -.-